app-fallbacks.js

```diff
@@ -74,6 +74,17 @@ function serveNodeVersionFallback(app) {
   return listenerHttpServer;
 }
 
+function serveDatabaseIssueFallbackRoutes(app) {
```

Comment:
Same idea for the database fallback: we dont have to boot up anything else for tests.